src/screens/Profile/Header/Metrics.tsx

```diff
@@ -47,17 +56,18 @@ export function ProfileHeaderMetrics({
         testID="profileHeaderFollowsButton"
         style={[a.flex_row, t.atoms.text]}
         to={makeProfileLink(profile, 'follows')}
-        label={_(msg`${following} following`)}>
+        label={_(msg`${following} following`)}
+        title={followingFull}>
         <Text style={[a.font_bold, a.text_md]}>{following} </Text>
         <Text style={[t.atoms.text_contrast_medium, a.text_md]}>
           {pluralizedFollowings}
         </Text>
       </InlineLinkText>
-      <Text style={[a.font_bold, t.atoms.text, a.text_md]}>
-        {formatCount(i18n, profile.postsCount || 0)}{' '}
-        <Text style={[t.atoms.text_contrast_medium, a.font_normal, a.text_md]}>
-          {plural(profile.postsCount || 0, {one: 'post', other: 'posts'})}
-        </Text>
+      <Text
+        style={[t.atoms.text_contrast_medium, a.text_md]}
+        title={postCountFull}>
+        <Text style={[t.atoms.text, a.font_bold, a.text_md]}>{postCount} </Text>
+        {pluralizedPostCount}
```

Comment:
I had to invert the nesting order of these <Text> elements so that the outer one was non-bold, and the inner one was bold, so that the tooltip would be non-bold when attached to the outer element. Hopefully I didn't break anything in the process.